lib/errors/error_handling.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module ErrorHandling
+  def perform_error_handling(error_record)
+    Rails.logger.info "Caught #{error_record.error}"
+    error_record.level = error_level(error_record.error)
+    if error_record.specific_error_tasks?
+      error_record.perform_error_handling
+    else
+      report_exception_sentry(error_record)
+    end
+    return nil
```

Comment:
Is this `return nil` needed?

Reply:
Nevermind, I guess you're explicitly stating that this doesn't return anything.

Reply:
Yes, you got it right, so that it doesn't return a `Raven` object. Earlier `WikiApi` and `Replica` both required this behaviour, now only `Replica` requires it. I thought it would be better to be explicit about it.